Documentation/contributing/development/dev_setup.rst

```diff
@@ -564,8 +564,20 @@ The tool ``cilium monitor`` can also be used to retrieve debugging information
 from the eBPF based datapath. Debugging messages are sent if either the
 ``cilium-agent`` itself or the respective endpoint is in debug mode. The debug
 mode of the agent can be enabled by starting ``cilium-agent`` with the option
-``--debug`` enabled or by running ``cilium config debug=true`` for an already
-running agent. Debugging of an individual endpoint can be enabled by running
+``--debug`` and ``--debug-verbose=datapath`` enabled or by running 
+``cilium config debug=true`` for an already running agent. The 
+``--debug-verbose=datapath`` option is necessary only if you want to trace the 
+`printk` messages in eBPF datapath. 
```

Comment:
Some elements are not clear from this modification.
- What is `--debug-verbose=datapath` needed for? I.e. what does it do that `--debug` does not?
- Does `cilium config debug=true` also set `--debug-verbose=datapath`, or does it need to be set separately?